sdk/test-utils/test-credential/review/test-credential.api.md

```diff
@@ -5,25 +5,17 @@
 ```ts
 
 import { AccessToken } from '@azure/core-auth';
-import { ClientSecretCredential } from '@azure/identity';
-import { TokenCredential } from '@azure/core-auth';
-import { TokenCredentialOptions } from '@azure/identity';
+import { DefaultAzureCredentialClientIdOptions } from '@azure/identity';
+import { DefaultAzureCredentialOptions } from '@azure/identity';
+import { DefaultAzureCredentialResourceIdOptions } from '@azure/identity';
+import { TokenCredential } from '@azure/identity';
```

Comment:
Can we use the the same `TokenCredential` import (either from Identity or core-auth) in both places so that we don't get this duplication?